content/docs/rfcs/71/README.md

```diff
@@ -0,0 +1,552 @@
+---
+slug: 71
+title: 71/STATUS-PUSH-NOTIFICATION-SERVER
+name: Push Notification Server
+status: raw
+category: Standards Track
+editor: Jimmy Debe <jimmy@status.im>
+contributors: 
+  - Andrea Maria Piana <andreap@status.im>
+---
+
+# Abstract
+Push notification server implementation for Android and iOS devices. This specification provides a set of methods that allow clients to use push notification services in mobile environments.
+
+# Background
+Push notification for iOS and Android devices can only be implemented by relying on [APN](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1), Apple Push Notification, service for iOS or [Firebase](https://firebase.google.com/) for Android. 
+
+For some Android devices, foreground services are restricted, requiring a user to grant authorization to applications to use foreground notifications. 
+Apple iOS devices restrict notifications to a few internal functions that every application can not use. 
+Applications on iOS can request execution time when they are in the background. This has a limited set of use cases for example, it will not schedule any time if the application was closed with force quit. 
+Requesting execution time is not responsive enough to implement a push notification system. 
+Status provides a set of methods to acheive push notification services.
+
+Since this can not be safely implemented in a privacy-preserving manner, clients need to be given an option to opt-in to receive and send push notifications. They are disabled by default.
+
+# Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### Definitions:
+
+client 
+> A node that implements the [Status specification](https://rfc.vac.dev/spec/6/). 
+
+user
+> The owner of a device that runs a client.
+
+server
+> A service that performs push notifications.
+
+Waku-Store  
+> A Waku node that decides to provide functionality to store messages permanently and deliver the messages to requesting clients. Follows [WAKU-STORE](https://rfc.vac.dev/spec/13/) specification
+
+
+### Components:
+
+gorush Instance 
+> Only used by push notification servers and MUST be publicly available.<br />
+
+Push Notification Server
+> Used by clients to register for receiving and sending notifications.<br />
+
+Registering Client
+> A client that wants to receive push notifications.<br />
+
+Sending Client
+> A client that wants to send push notifications.<br />
+
+Requirements:<br />
```

Comment:
Nitpick: do we want this BR tag here?

Reply:
Thanks for the feedback, I changed definitions and components to a table, it does look better this way.  Added a link for SHAKE-256 from NIST. Also removed unnecessary BR tags.